demos-europe/demosplan-core
# Feat(refs:T31913): Hide press announcement if no press news existing

**Ticket:** https://yaits.demos-deutschland.de/T31913

### Description: 

- press news section shall be hidden for all projects, if no press news existing
- in this the width of the global news get added by the width of the press announcement section

### How to review/test

- go to startpage (you dont need to login)
- go to the section `Aktuelle Meldungen` and click to `Alle Meldungen`
- if press news not existing you will only see the global news
- if you have no press news activated, login as `Mandanten-Administration`and click to `Portal-Mitteilungen`
  -> here you can activate press news to show it in `Alle Meldungen` or create a new one
  -> than logout and go again to the startpage to `Alle Meldungen`

### Linked PRs (optional)

- https://github.com/demos-europe/demosplan-project-diplanbau/pull/80
- https://github.com/demos-europe/demosplan-project-blp/pull/38

### PR Checklist
